### DISCUSSION TITLE
Setting up Bitcoin Core for Runes?

### DISCUSSION THREAD

**Discussion by SetsuAnt** [ordinals/ord#3528] [Q&A]
Is there a guide for installing the Rune software on Bitcoin Core/Ord? And is there also a list of commands for buying/minting runes? Thanks!

**Comment by raphjaph** [comment]
docs.ordinals.com

**Comment by cbspears** [comment]
docs.ordinals.com is the best place. If you want some unofficial community guides and a great community resource, check out the ordicord: https://discord.gg/ordinals
